Uanion/Uanion.Application/MappingProfiles/UserMappingProfile.cs

```diff
@@ -1,15 +1,17 @@
-﻿using AutoMapper;
-using Uanion.Application.Features.User.Commands.CreateUser;
+﻿using Uanion.Application.Features.User.Commands.CreateUser;
+using Uanion.Application.Features.User.Commands.UpdateUser;
 using Uanion.Application.Features.User.Queries.GetUser;
 using Uanion.Domain.Entities;
 
-namespace Uanion.Application.Profiles;
+namespace Uanion.Application.MappingProfiles;
 
 public class UserMappingProfile : AutoMapper.Profile
 {
     public UserMappingProfile()
     {   
         CreateMap<CreateUserCommand, User>();
+        CreateMap<UpdateUserCommand, User>();
+
```

Comment:
Need to register mapper between `User` and `UserListViewModel` .

Reply:
Done